examples/viewer.js

```diff
@@ -264,3 +373,233 @@ function sendViewerMessage(message) {
+        remoteCandidate = stats.get(activeCandidatePair.remoteCandidateId);
+    }
+
+    // Capture the IP and port of the remote candidate
+    if (remoteCandidate) {
+        if (remoteCandidate.address && remoteCandidate.port) {
+            connectionString = remoteCandidate.address + ' : ' + remoteCandidate.port + ' - ' + remoteCandidate.protocol;
+        } else if (remoteCandidate.ip && remoteCandidate.port) {
+            connectionString = remoteCandidate.ip + ' : ' + remoteCandidate.port + ' - ' + remoteCandidate.protocol;
+        } else if (remoteCandidate.ipAddress && remoteCandidate.portNumber) {
+            connectionString = remoteCandidate.ipAddress + ' : ' + remoteCandidate.portNumber + ' - ' + remoteCandidate.protocol;
+        }
+    }
+
+    if(activeCandidatePair) {
+        
+        // Get the current RTT the pair.
+        rttCurrent = activeCandidatePair.currentRoundTripTime;
+
+        //Get the video stats.
+        stats.forEach(report => {
+            if (report.type == 'inbound-rtp' && report.mediaType == 'video') {
+                videoFramerate = report.framesPerSecond;
+                videoHeight = report.frameHeight;
+                videoWidth = report.frameWidth;
+                videojitter = report.jitter;
+                videoDecodedFrameCount = report.framesDecoded;
+                videoDroppedFrameCount = report.framesDropped;
+
+                const bytes = report.bytesReceived;
+                if(vTimeStampPrev) {
+
+                    videoBitrate = 8 * (bytes - vBytesPrev) / (report.timestamp - vTimeStampPrev);
+                    videoBitrate = Math.floor(videoBitrate);
+                    curDroppedFrames = videoDroppedFrameCount - vFDroppedPrev;
+
+
+                }
+                vBytesPrev = bytes;
+                vTimeStampPrev = report.timestamp;
+                vFDroppedPrev = videoDroppedFrameCount;
+            }
+        });
+
+        //Get the audio stats.
+        stats.forEach(report => {
+            if (report.type == 'inbound-rtp' && report.mediaType == 'audio') {
+                audiojitter = report.jitter;
+                audioSamplesReceived = report.totalSamplesReceived;
+
+                const bytes = report.bytesReceived;
+                if(aTimeStampPrev) {
+                    audioBitrate = 8 * (bytes - aBytesPrev) / (report.timestamp - aTimeStampPrev);
+                    audioBitrate = Math.floor(audioBitrate);
+                }
+                aBytesPrev = bytes;
+                aTimeStampPrev = report.timestamp;
+
+            }
+        });
+
+        // Only start test and display metrics once something has been decoded
+        if ((videoDecodedFrameCount > 0) || (audioSamplesReceived > 0)){
+
+            var currentDate = new Date();
+            var currentTime = currentDate.getTime();
+
+            //timestamp start of decoded frames
+            if (statStartTime == 0) { 
```

Comment:
Comparison statStartTime == 0 may cause unexpected type coercion, can we use === instead? (same in a few other places where == is used e.g. line 469)

Reply:
fixed

Reply:
Few other spots: 409, 445, and 467

Reply:
fixed the rest with the last push